dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/GitHubHelperTest.java

```diff
@@ -0,0 +1,32 @@
+package io.dockstore.webservice.helpers;
+
+import java.io.IOException;
+
+import io.dockstore.webservice.core.LicenseInformation;
+import org.junit.Assert;
+import org.junit.Test;
+import org.kohsuke.github.AbuseLimitHandler;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+import org.kohsuke.github.RateLimitHandler;
+
+public class GitHubHelperTest {
+
+    // TODO: Replace repos with something less likely to change
+    @Test
+    public void testGitHubLicense() throws IOException {
+        GitHub gitHub = new GitHubBuilder().withRateLimitHandler(RateLimitHandler.WAIT).withAbuseLimitHandler(
+                AbuseLimitHandler.WAIT).build();
+        LicenseInformation licenseInformation = GitHubHelper.getLicenseInformation(gitHub, "dockstore-testing/md5sum-checker");
```

Comment:
Interesting, hadn't noticed/clicked until now that you're doing this for the entry, not the version. This means it's getting the license for the repo. Is it going off the default branch? Probably fine for now, but I wonder if we eventually need to track at the version level. It's rare that licenses change, although I've seen it, and I don't know if they're retroactive or not -- I guess it depends on how the license content.

Reply:
GitHub doesn't track it at version level